config/config.go

```diff
@@ -58,28 +60,30 @@ func get() (*Config, error) {
 	}
 
 	cfg := &Config{
-		APIRouterURL:                   "http://localhost:23200/v1",
-		BindAddr:                       "localhost:25000",
-		CacheCensusTopicUpdateInterval: 30 * time.Minute,
-		CacheNavigationUpdateInterval:  30 * time.Minute,
-		CensusTopicID:                  "4445",
-		Debug:                          false,
-		DefaultLimit:                   10,
-		DefaultMaximumLimit:            50,
-		DefaultMaximumSearchResults:    500,
-		DefaultOffset:                  0,
-		DefaultPage:                    1,
-		DefaultSort:                    "relevance",
-		EnableCensusTopicFilterOption:  false,
-		EnableNewNavBar:                false,
-		GracefulShutdownTimeout:        5 * time.Second,
-		HealthCheckCriticalTimeout:     90 * time.Second,
-		HealthCheckInterval:            30 * time.Second,
-		IsPublishing:                   false,
-		NoIndexEnabled:                 false,
-		ServiceAuthToken:               "",
-		SiteDomain:                     "localhost",
-		SupportedLanguages:             []string{"en", "cy"},
+		APIRouterURL:                            "http://localhost:23200/v1",
+		BindAddr:                                "localhost:25000",
+		CacheCensusTopicUpdateInterval:          30 * time.Minute,
+		CacheNavigationUpdateInterval:           30 * time.Minute,
+		CensusTopicID:                           "4445",
+		Debug:                                   false,
+		DefaultLimit:                            10,
+		DefaultMaximumLimit:                     50,
+		DefaultMaximumSearchResults:             500,
+		DefaultOffset:                           0,
+		DefaultPage:                             1,
+		DefaultSort:                             "relevance",
+		EnableCensusTopicFilterOption:           false,
+		EnableCensusPopulationTypesFilterOption: false,
+		EnableCensusDimensionsFilterOption:      false,
+		EnableNewNavBar:                         false,
```

Comment:
Sorry to pedantic but everything else is in alphabetical order so would be best to follow the convention re `EnableCensusPopulationTypesFilterOption` and `EnableCensusDimensionsFilterOption`